harvester/harvester-installer
# Disk page in Installer requires pressing Enter two times before switching to next page

**Problem:**
After the user entered all information in the disk page it is still necessary to press `Enter` or `Key-Down` twice at the `Use MBR partitioning scheme` form control before switching to the next (Hostname) page.

**Solution:**
Set the `diskConfirmed` after the `Persistent size` has been validated.

**Related Issue:**
https://github.com/harvester/harvester/issues/5097

**Test plan:**
- Go to the disk page.
- Enter all data, so that the form is valid.
- Navigate to `Use MBR partitioning scheme`. Press `Enter` or `Key-Down`.
- The page should switch to the `Hostname` page.
